AnnoyancesFilter/sections/popups.txt

```diff
@@ -121,6 +121,7 @@ quirkbooks.com#$?#html { overflow: visible !important; touch-action: auto !impor
 bing.com##.ev_talkbox_wrapper
 gizbot.com###containerPop
 wikihow.com###contribute_footer_wrap
+telerik.com###footer > div.feedback-row
```

Comment:
I don't think that we should block it. It's a useful element, especially when you press `No`.

Reply:
@zloyden, i thought so too, but in same cases it gets fixed to the bottom and becomes pretty annoying.

Reply:
I think it can be blocked.